testsuite/MDAnalysisTests/analysis/test_contacts_best.py

```diff
@@ -0,0 +1,114 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
How about merging the test with `test_contacts.py` because then it is quite clear that there's one test file for one module.